docs/quickstarts/nextjs.mdx

```diff
@@ -145,6 +145,19 @@ export const config = {
 Clerk offers a set of [prebuilt components](/docs/components/overview) that can gate access to content based on whether a user is signed in and/or has the correct permissions. You can use Clerk's [`<SignedIn>`](/docs/components/control/signed-in) and [`<SignedOut>`](/docs/components/control/signed-out) components to restrict access to content within pages and even whole pages. The `<SignedIn>` component displays the content inside the tags only when the user is signed in. The `<SignedOut>` components displays its content only when the user is signed out.
 
 
+Clerk provides prebuilt components that control content access based on user authentication and permissions. These components can be used to restrict access to specific content or entire pages.
+To manage content visibility for signed-in users, use the `<SignedIn>` component:
+- Enclose content within the `<SignedIn>` tags to make it viewable only by users who are currently signed in.
+- Content outside of these tags remains accessible to both signed-in and signed-out users.
+To manage content visibility for signed-out users, use the `<SignedOut>` component:
+- Enclose content within the `<SignedOut>` tags to make it visible only to users who are currently not signed in.
+- Content outside of these tags is only accessible to signed-in users.
+For more information on these components, refer to the following documentation:
+- Overview of Clerk's Prebuilt Components: /docs/components/overview
+- Control Access with `<SignedIn>`: /docs/components/control/signed-in
+- Control Access with `<SignedOut>`: /docs/components/control/signed-out

```

Comment:
I agree with this sentiment, the rest looks great!! 

https://github.com/clerk/clerk-docs/pull/649#discussion_r1457945504